metaflow/plugins/kfp/kfp.py

```diff
@@ -1,8 +1,9 @@
 import inspect
```

Comment:
I think we need to add additional tests into the integration testing framework to test this PR? The existing tests don't have anything to do with KFP components binding (they're just pure Metaflow code samples).

Reply:
Will be covered in [AIP-2983](https://zbrt.atl.zillow.net/browse/AIP-2983).

Reply:
Thanks!